cfme/common/provider.py

```diff
@@ -92,6 +92,13 @@ def __init__(self, **kwargs):
                 self.token = kwargs['token']
 
     @property
+    def properties_form(self):
```

Comment:
Well understand, but, ```return self._properties_form.properties_form``` isn't something people are going to have to use, though I do agree it doesn't look nice, I can change it to properties_region...... @mfalesni agreement from you here too?